src/pre-core-exemptions/README.md

```diff
@@ -0,0 +1,53 @@
+# Pre Core Exemptions
+
+This package serves as a way for users deploying uds-core to pass exemptions to core for things that are deployed before core.
+
+For example, if a team is deploying a bundle containing a custom init package that has rook-ceph, the first time it
+deploys everything will be fine, but if it cycles for whatever reason once core is deployed then rook-ceph will be denied by Pepr policies. Thus an exemption CR is needed.
+The problem, though, is the init package can't deploy an exemption resource when the exemption CRD has not yet been deployed by core. 
```

Comment:
```suggestion
For instance, a team is deploying a bundle that includes a custom init package with rook-ceph. During install, everything goes smoothly since Pepr's validation is not present. However, during an upgrade after the core deployment, Pepr policies might deny new rook-ceph pods. 
```

bit of a nit but this wording was hard for me. slight chatgpt on this so feel free to tweak.